app/models/enum/geostore.py

```diff
@@ -4,3 +4,18 @@
 class GeostoreOrigin(str, Enum):
     gfw = "gfw"
     rw = "rw"
+
+
+class LandUseType(str, Enum):
+    fiber = "fiber"
+    logging = "logging"
+    # mining = "mining"  # Present in the docs, but yields a 404
+    oilpalm = "oilpalm"
+    tiger_conservation_landscapes = "tiger_conservation_landscapes"
+
+
+class LandUseTypeUseString(str, Enum):
+    fiber = "gfw_wood_fiber"
+    logging = "gfw_logging"
+    oilpalm = "gfw_oil_palm"
+    tiger_conservation_landscapes = "tcl"
```

Comment:
Having `enum`s for land use will not work since the param is [open-ended in the gfw-geostore-api microservice](https://github.com/gfw-api/gfw-geostore-api/blob/a6cc3432c90d316106af8af7e90c362f9cc5d7b8/app/src/routes/api/v2/geoStore.router.js#L219).

I quick look at the geostore logs for the past 10 days shows that `default` is frequently used with values such as:
- `river_basins`
- `biodiversity_hotspots`
- `endemic_bird_areas`

to name a few.

Reply:
Thanks for re-discovering/remembering this! It's fixed now, I'm just passing through the string for land_use as-is. Just tested with these examples in dev!